src/DataImport/gb_parse.py

```diff
@@ -26,6 +27,8 @@ def parse_genbank(genbank_file):
                 # uses filename (without extension) as species name
                 species = os.path.splitext(os.path.basename(genbank_file)[0])
 
+            print species
```

Comment:
Why are you printing the species here? Is this just to keep track of what's going on or does it serve a larger purpose?